test/WebJobs.Script.Tests/UtilityTests.cs

```diff
@@ -353,5 +354,22 @@ public void IsHttporManualTriggerTests(string triggerType, bool expectedResult)
         {
             Assert.Equal(expectedResult, Utility.IsHttporManualTrigger(triggerType));
         }
+
+        [Theory]
+        [InlineData("createIsolationEnvironment", true)]
+        [InlineData("HttpTrigger2", true)]
+        [InlineData("HttptRIGGER", true)]
+        [InlineData("MANUALtRIGGER", true)]
+        [InlineData("Function-200", true)]
+        [InlineData("hello!", false)]
+        [InlineData("🙅", false)]
+        public void IsValidFunctionNameTests(string functionName, bool expectedResult)
+        {
+            CultureInfo defaultCulture = Thread.CurrentThread.CurrentCulture;
+            // In Turkish-Turkey, "I" is not the uppercase equivalent of "i"
+            Thread.CurrentThread.CurrentCulture = new CultureInfo("tr-TR");
```

Comment:
Agree with the comment above. Instead of always setting the culture here, pass that in with different tests